admin/class-admin.php

```diff
@@ -18,23 +16,22 @@ class Admin {
 	 * Class constructor.
 	 */
 	public function __construct() {
-		$this->init();
```

Comment:
Moved these to public functions to stay in keeping with only using constructor for variable inits.